pixel/theme.json

```diff
@@ -369,32 +371,34 @@
 			},
             "h1": {
                 "typography": {
-                    "fontSize": "3rem"
+                    "fontSize": "clamp(1.375rem, 1.2916666666666667rem + 0.4166666666666667vw, 1.625rem)"
                 }
             },
             "h2": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--x-large)"
+                    "fontSize": "clamp(1.25rem, 1.2083333333333333rem + 0.20833333333333334vw, 1.375rem)"
                 }
             },
             "h3": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--large)"
+                    "fontSize": "clamp(1.125rem, 1.0833333333333333rem + 0.20833333333333334vw, 1.25rem)"
                 }
             },
             "h4": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--medium)"
+                    "fontSize": "1.125rem"
                 }
             },
             "h5": {
                 "typography": {
-                    "fontSize": "1.125rem"
+                    "fontSize": "0.813rem",
+                    "textTransform": "uppercase"
                 }
             },
             "h6": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--small)"
+                    "fontSize": "0.75rem",
+                    "textTransform": "uppercase"
```

Comment:
update the font weight of `heading` to `500` and override it to `400` in `h6`

Reply:
Done in https://github.com/Automattic/themes/pull/6479/commits/b4d39e520493859d023dff9835254f388d6fa2b6.